profiles/services.py

```diff
@@ -708,9 +742,10 @@ def calculate_performance_values(
                 name=name,
                 value=value,
             )
+            for name, value in result.performance_values.items()
```

Comment:
somehow missed this when fixing https://github.com/Syriiin/osuchan-backend/pull/65, but it didn't actually cause any issues so no harm no foul?